libr/asm/arch/arm/armass.c

```diff
@@ -421,6 +421,10 @@ static int thumb_assemble(ArmOpcode *ao, ut64 off, const char *str) {
 	} else
 	if (!strcmpnull (ao->op, "b") || !strcmpnull (ao->op, "b.n")) {
 		//uncond branch : PC += 4 + (delta*2)
+		int offset = getnum (ao->a[0]);
+		if (offset == 0) {
```

Comment:
oh! but thats not correct, because `b 0` is vaild. getnum should return -1 in case of fail, isnt?